release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -259,9 +274,8 @@ submodule openconfig-bgp-common {
         "Enable route flap damping.";
     }
 
-    leaf send-community {
+    leaf-list send-community {
       type oc-bgp-types:community-type;
-      default "NONE";
```

Comment:
Please make this a non-breaking change by marking 'send-community' as deprecated and introduce a new leaf list, for example `send-community-types`.    This example name also clarifies that the data is a community type, not a list of community values.